content/02.main-text.md

```diff
@@ -19,3 +19,24 @@ We wrote entirely in the open without restrictions on who was welcome to contrib
 Although we requested that some authors participate for their specific expertise, most discovered the manuscript organically through conferences or social media and independently decided to contribute.
 `TODO: confirm "most"`
 To coordinate this effort, we developed a manuscript writing process using the Markdown language, the GitHub software development platform [@url:https://github.com/greenelab/deep-review/], and our new Manubot tool [@url:https://github.com/greenelab/manubot-rootstock @url:https://github.com/greenelab/manubot] for automating manuscript generation.
+
+
+### Authorship
+
+To determine authorship we followed the International Committee of Medical Journal Editors guidelines, which recommend authors substantially contribute to, draft, approve, and agree to be accountable for the manuscript [@url:http://www.icmje.org/recommendations/browse/roles-and-responsibilities/defining-the-role-of-authors-and-contributors.html].
```

Comment:
Do we want to note that we supplemented that by requiring attributable contributions on github?